_books/ion-1-1/src/text/values.md

```diff
@@ -0,0 +1,489 @@
+# Values
+
+## Annotations
+In the text format, type annotations are denoted by a [symbol token](symbol-tokens.md) and double-colons preceding any value.
+Multiple annotations on the same value are separated by double-colons:
+
+```ion
+int32::12                                // Suggests 32 bits as end-user type
+degrees::'celsius'::100                  // You can have multiple annotaions on a value
+'my.custom.type'::{ x : 12 , y : -1 }    // Gives a struct a user-defined type
+
+{ field: some_annotation::value }        // Field's name must precede annotations of its value
+
+jpeg :: {{ ... }}                        // Indicates the blob contains jpeg data
+bool :: null.int                         // A very misleading annotation on the integer null
+'' :: 1                                  // An empty annotation
+null.symbol :: 1                         // ERROR: type annotation cannot be null 
```

Comment:
Should add ERROR examples for annotations on an e-expression and expression group.